src/calculateRentalCost.js

```diff
@@ -20,7 +21,13 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  if (days === 7 || days > 7) {
+    return days * 40 - 50;
+  } else if (days === 3 || days === 4 || days === 5 || days === 6) {  // (days === 3 || days > 3 && days < 7) - eslint doesn't like this
```

Comment:
зачем перечислять каждый день?
```suggestion
  } else if (days >= 3 && days <= 6) {
```